draft-ietf-scitt-architecture.md

```diff
@@ -547,18 +547,17 @@ It indicates the Signed Statement is securing a JSON content type, and identifyi
 
 To register a Signed Statement, the Transparency Service performs the following steps:
 
-1. **Client authentication:** A Client authenticates with the Transparency Service, to Register Signed Statements on behalf of one or more Issuers.
-Authentication and authorization is implementation-specific, and out of scope of the SCITT Architecture.
-1. **Issuer Verification:** The Transparency Service MUST syntactically validate the Issuer's identity Claims, which may be different than the Client identity.
+1. **Client authentication:** A Client authenticates the Transparency Service before registering Signed Statements on behalf of one or more Issuers.
+Authentication and authorization is implementation-specific, and out of scope of the SCITT architecture.
+1. **Issuer Verification:** The Transparency Service MUST validate the Issuer's identity Claims, which may be different than the Client identity.
```

Comment:
'syntactically' was a hard-fought compromise in a long discussion about how far down the stack of turtles an implementation should have to go in order to validate identity claims. For example, a signature should be well-formed and locally verify the signature against the protected public key, but whole chains, CRLs, OCSP etc checks do not need to be done (necessarily).

Reply:
If it means a lot to you then leave the syntactical term in there. I am just pointing out that the actual issuer verification is more than just a syntax check since it really requires cryptographic verification.

Reply:
Constraining the verification requirement to depend only on information available directly in-line with the claims was the desired effect. That principal means a lot, the specific word 'syntactic' does not.

Reply:
@robinbryce Using only information from the COSE headers is fine. Like you, I would call this "syntactic"

Reply:
Any suggestion on what to do about the term 'syntactically' now? @robinbryce @JAG-UK

Reply:
I think what is important is trusting the issuer's keys... the claims used to discover them are hints.

Reply:
LGTM to latest changes